spell/rwa-onboarding-checklist.md

```diff
@@ -11,64 +12,73 @@
+        * [ ] `vat` matches `MCD_VAT`
+        * [ ] `jug` matches `MCD_JUG`
+        * [ ] `gemJoin` matches `MCD_JOIN_RWAXXX_A` (Per spell code)
+        * [ ] `daiJoin` matches `MCD_JOIN_DAI`
+        * [ ] `outputConduit` matches `RwaOutputConduit<2,3>` (Per spell code)
       * [ ] check `wards`
         * [ ] `MCD_PAUSE_PROXY` is relied
+        * [ ] `MCD_ESM` is being relied in the spell (as approved by GovAlpha)
         * [ ] deployer is denied
         * [ ] no other address has been relied
     * [ ] `RwaJar`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
         * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
-      * [ ] constructor arg matches [chainlog](https://chainlog.makerdao.com/)
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/jars/RwaJar.sol) and it's consistent with previous RWA onboarding
+      * [ ] constructor arg matches [`ChainLog`](https://chainlog.makerdao.com/)
         * [ ] `chainlog`
       * [ ] no `wards`
     * [ ] `RwaInputConduit<2,3>`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
         * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/conduits/RwaSwapInputConduit2.sol) and it's consistent with previous RWA onboarding
       * [ ] constructor args are correct and match [chainlog](https://chainlog.makerdao.com/)
         * [ ] `dai`
         * [ ] `to` (`RwaUrn`, `RwaJar`)
         * [ ] `gov` (`v1`)
         * [ ] `psm`, `gem` (`v3`)
       * [ ] check `wards`
         * [ ] `MCD_PAUSE_PROXY` is relied
+        * [ ] `MCD_ESM` is being relied in the spell (as approved by GovAlpha)
         * [ ] deployer is denied
         * [ ] no other address has been relied
     * [ ] `RwaOutputConduit<2,3>`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
         * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/conduits/RwaSwapOutputConduit.sol) and it's consistent with previous RWA onboarding
```

Comment:
I put the commit hash here since it's what we are using and makes it easier for reviewers and crafters to check the source.

Otherwise you'd have to find the correct version every spell; you don't need to use the commit hash, but it's there if you want it.

Reply:
BTW new conduits are being audited, so this shouldn't be an issue.